packages/moonstone/DatePicker/DatePickerBase.js

```diff
@@ -248,11 +288,14 @@ const DatePickerBase = kind({
 						const isLast = index === order.length - 1;
 						const isLeft = isFirst && !rtl || isLast && rtl;
 						const isRight = isFirst && rtl || isLast && !rtl;
+
 						switch (picker) {
 							case 'd':
 								return (
 									<DateComponentRangePicker
+										aria-label={ariaLabel}
```

Comment:
I don't think I was clear on my intent. I wouldn't expect to use the same label for all pickers. Instead, this would use `dayAriaLabel` and we'd used `dayAccessibilityHint` (a new prop) below.

Reply:
I will apply this.